Dockerfile

```diff
@@ -37,6 +37,7 @@ RUN \
     rsync \
     wget \
     which \
+    xfsprogs \
```

Comment:
I'd prefer to put this in the "sdk-plus" section, to keep better track of what's needed for building software in the SDK (this section) vs what's needed for building Bottlerocket (that section).

Reply:
Works for me, I really just sort of threw it in and hoped for success. I'll move it to that section.